pallets/parachain-system/src/unincluded_segment.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright 2023 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+use codec::{Decode, Encode};
+use cumulus_primitives_core::{relay_chain, ParaId};
+use scale_info::TypeInfo;
+use sp_std::collections::btree_map::BTreeMap;
+
+pub struct BandwidthLimits {}
```

Comment:
Not all limits mentioned in the issue are available to parachains. AbridgedHostConfiguration only includes per-candidate limit for hrmp, this struct can only target (as of now) aggregated changes for ump queue.

Reply:
Note that the `HrmpChannel` structs which are contained in the relay-chain storage proof has current sizes and limits contained within them.